gateway/conf/deployment.yaml

```diff
@@ -36,16 +36,17 @@ ballerina.config:
             packageName: org.wso2.carbon.apimgt.gateway.auth
             # Ballerina Archive name.
             archiveName: auth.bmz
-          -
-            # Package name of the request interceptor.
-            packageName: org.wso2.carbon.apimgt.gateway.throttle
-            # Ballerina Archive name.
-            archiveName: throttle.bmz
-          -
-            # Package name of the request interceptor.
-            packageName: org.wso2.carbon.apimgt.gateway.handler
-            # Ballerina Archive name.
-            archiveName: analytics.bmz
+#          -
+#            # Package name of the request interceptor.
+#            packageName: org.wso2.carbon.apimgt.gateway.throttle
+#            # Ballerina Archive name.
+#            archiveName: throttle.bmz
+#          -
+#            # Package name of the request interceptor.
+#            packageName: org.wso2.carbon.apimgt.gateway.handler
+#            # Ballerina Archive name.
+#            archiveName: analytics.bmz
+
```

Comment:
It is better to remove those commented configurations. We can update the docs on how to configure deployment.yaml when starting as the offline gateway.

Reply:
noted